src/redux/slices/auth.ts

```diff
@@ -0,0 +1,28 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  isSignedIn: false,
+  username: '',
+};
+
```

Comment:
Ông lên dùng một biến enum ở đây cho dễ nhận biết hơn

```suggestion
export enum LoginStatus {
      Success = 'Login Success',
      Failed = 'Login Failed',
      None = 'None',
},

export enum LogoutStatus {
      Success = 'Logout Success',
      Failed = 'Logout Failed',
      None = 'None',
},
```

Reply:
tôi có dùng api hẳn hoi đâu mà làm rõ đến mức đó

Reply:
Ông chat mà cũng ngọng thì chịu